### PR TITLE
Added footer

### PR DESCRIPTION
**Pull request by ankitbajpai65**
# Zero To Mastery - Webblocks 2022 📦📦

I have added a footer built using HTML, CSS, and vanilla javascript.
image

### CODE REVIEW

File: src/blocks/ankitbajpai65/index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                    </ul>
+                </div>
+                <div class="footerPart">
+                    <h3>GET IN TOUCH</h3>
+                    <p>Any questions? Let us know in store at BP 473 Complexe universitaire Al Qods, Oujda 60000 or call
+                        us
+                        on (+212) 65964665</p>
+                    <div class="imgDiv">
+                        <span><img src="icons\fb.png" alt=""></span>
+                        <span><img src="icons\instagram.png" alt=""></span>
```
Comment (sophiabrandt) [inline src/blocks/ankitbajpai65/index.html:41]: This won't work on an Unix system, you need to change it to `icons/fb.png`.

File: src/blocks/ankitbajpai65/style.css [hunk 2]
```diff
@@ -0,0 +1,100 @@
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+#mainDiv {
+    /* border: 2px solid red; */
```
Comment (sophiabrandt) [inline src/blocks/ankitbajpai65/style.css:8]: Please remove outdated code.

File: src/blocks/ankitbajpai65/style.css [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+}
+
+
+.footer {
+    background-color: black;
+    height: 20rem;
+}
+
+.footerDiv {
+    /* border: 2px solid black; */
```
Comment (sophiabrandt) [inline src/blocks/ankitbajpai65/style.css:21]: Same here.

File: src/blocks/ankitbajpai65/style.css [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+    height: 86%;
+    width: 100%;
+    display: flex;
+}
+
+.footerPart {
+    /* border: 2px solid red; */
+    /* background-color: black; */
```
Comment (sophiabrandt) [inline src/blocks/ankitbajpai65/style.css:29]: Here

File: src/blocks/ankitbajpai65/style.css [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+    width: 25%;
+    padding-top: 3rem;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    /* justify-content: center; */
```
Comment (sophiabrandt) [inline src/blocks/ankitbajpai65/style.css:35]: Here

File: src/blocks/ankitbajpai65/index.html [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+                        <li><a href="">Help4</a></li>
+                    </ul>
+                </div>
+                <div class="footerPart">
+                    <h3>GET IN TOUCH</h3>
+                    <p>Any questions? Let us know in store at BP 473 Complexe universitaire Al Qods, Oujda 60000 or call
+                        us
+                        on (+212) 65964665</p>
+                    <div class="imgDiv">
+                        <span><img src="icons\fb.png" alt=""></span>
```
Comment (sophiabrandt) [inline src/blocks/ankitbajpai65/index.html:40]: Please provide the license/source link for the Images to make sure that you have the right to use those.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]

File: src/blocks/ankitbajpai65/index.html (hunk 1 above)
Comment (sophiabrandt) [reply src/blocks/ankitbajpai65/index.html:41]: All other images, too.

File: src/blocks/ankitbajpai65/style.css (hunk 5 above)
Comment (ankitbajpai65) [reply src/blocks/ankitbajpai65/style.css]: Hi there, I removed the comments/outdated code from CSS file as you mentioned. Please check the same.
Thanks.

File: src/blocks/ankitbajpai65/index.html [hunk 7]
```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
```
[the remark this answers is not in the document]
Comment (ankitbajpai65) [reply src/blocks/ankitbajpai65/index.html]: I changed the title to an HTML file. Please check the same.
Thanks.

**Comment by ankitbajpai65** [comment]
Hello @sophiabrandt, I have performed all the changes you mentioned above. Kindly check it and please let me know if anything required.
Thanks.

**Pull request merged**